app/models/concerns/bank_account_validations.rb

```
@@ -0,0 +1,112 @@
+
+module BankAccountValidations
+  extend ActiveSupport::Concern
+
+  included do
+    alias_attribute :bank_code, :bank_number
+
+    validate :validate_bank_code
+    validate :validate_branch_code
+    validate :validate_account_number, if: -> {
+      production_only_models = ["JordanBankAccount", "BahrainBankAccount", "TunisiaBankAccount", "OmanBankAccount",
+                               "NorwayBankAccount", "CzechRepublicBankAccount", "GuatemalaBankAccount", "NigerBankAccount",
```

RuboCop (1.75.5)

[convention] Align the elements of an array literal if they span more than one line.

(Layout/ArrayAlignment)